### ISSUE TITLE
Model.eval() while training

### ISSUE THREAD

**Issue by Esaada** [facebookresearch/FixRes#9]
Hi I'm looking at this file:
https://github.com/facebookresearch/FixRes/blob/master/imnet_finetune/train.py
And I saw that in the training loop you always do: self._state.model.**eval().**
Why is that?
 And why do you change from eval() to train() only in :
 self._state.model.module.layer4[2].bn3.train()
and 
self._state.model.module.layer4[2].bn3.eval()

I never seem anything like that, I would be grateful for explanation.

**Comment by TouvronHugo** [comment]
Hi @BarakBat ,
We put the model in eval mode and we use self._state.model.module.layer4[2].bn3.train() so that during fine-tuning the batch norm remains fixed except the last batch norm (this compensates the statistical change in the network due to the resolution change of the images). This is further detailed in the article.
